### PR TITLE
VACMS 18909 - KISS selectors for VA Healthcare Services from content-build

### PR DESCRIPTION
**Pull request by eselkin**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- Creates the selectors for the `/data/cms/va-healthcare-services.json` data
- Sitewide team

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18909
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18910

## Testing done

- No prior behavior.
- Tests were added on whether selector functions to retrieve data

## Screenshots

N/A

## What areas of the site does it impact?

No area affected, just selectors created

## Acceptance criteria

- [x] Selectors added to vets-website for array results of each type (VBA, Vet Cetners, VAMC, TRICARE)

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added (N/A)
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed (N/A)

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (unauthenticated)

## Requested Feedback

Check code and tests (no visible or console statements are visible)

### CODE REVIEW

File: src/platform/site-wide/drupal-static-data/source-files/va-health-services/connect/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { connectDrupalStaticDataFile } from '../../../connect';
+import { preProcessHealthServicesData } from './preProcess';
+
+export const connectDrupalStaticDataFileVamcEhr = dispatch => {
```
Comment (SnowboardTechie) [inline src/platform/site-wide/drupal-static-data/source-files/va-health-services/connect/index.js:4]: I believe we may want this const name adjusted to something like 
```suggestion
export const connectDrupalStaticDataFileVaHealthServices = dispatch => {
```

Comment (eselkin) [reply src/platform/site-wide/drupal-static-data/source-files/va-health-services/connect/index.js:4]: Oops. Absolutely we do.

Comment (eselkin) [reply src/platform/site-wide/drupal-static-data/source-files/va-health-services/connect/index.js]: Renamed, thanks!

**Review by SnowboardTechie — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**